### ISSUE TITLE
[pushover] Priority message documentation is misleading

### ISSUE THREAD

**Issue by jlaur** [openhab/openhab-addons#15733]
I did some testing before implementing new rules and got stuck for a while here:
```javascript
var receipt = action.sendPriorityMessage("Test", "Priority message", 1);
if (receipt === null) {
	console.log("Receipt is null");
	return;
}
console.log("receipt is: " + receipt);
cache.private.put("pushover_receipt", receipt);
setTimeout(() => {
	var receipt = cache.private.get("pushover_receipt");
	console.info("Cancelling: " + receipt);
	action.cancelPriorityMessage(receipt);
}, 10000);
```

This rule does not work - it cannot cancel the priority message, and the `receipt` is "".

The reason why I did this attempt was a misunderstanding from my side about what it means to "cancel" a message. I thought it would remove the notification, but actually it will just stop the "retry", i.e. sending the same notification to the user over and over again. And herein lies the problem: This only makes sense for emergency messages, i.e. priority 2.

This is documented here: https://pushover.net/api#priority

I ended up configuring the requests in Postman and could clearly see the difference between priority 1 and 2. For priority 1 there is no receipt in the response payload (request anonymized):

```json
{
    "status": 1,
    "request": "00000000-0000-0000-0000-000000000000"
}
```

This is documented here: https://pushover.net/api/receipts

Another issue is that the priorities are not documented. For example, 1 is "high priority" and 2 is "emergency priority". Therefore the comment below this table: https://www.openhab.org/addons/bindings/pushover/#thing-configuration

> The `retry` and `expire` parameters are only used for emergency-priority notifications.

lacks the context/explanation for what an emergency priority message is. Only documentation is: https://www.openhab.org/addons/bindings/pushover/#thing-actions

> The `sendPriorityMessage` action returns a `String` value (the `receipt`) if the message was sent successfully, otherwise `null`.
>
> - `sendPriorityMessage(String message, @Nullable String title, @Nullable Integer priority)` - This method is used to send a priority message. Parameter `priority` is the priority (`-2`, `-1`, `0`, `1`, `2`) to be used (default: `2`).

Summing up, to avoid confusion I suggest to:
- Add the meaning of the priority numbers.
- Mention that `receipt` is only applicable for emergency priority messages.
